cmd/bgp/descriptor/adapter/globalconf.go

```diff
@@ -0,0 +1,233 @@
+// Code generated by adapter-generator. DO NOT EDIT.
```

Comment:
This was probably unintended, but you should not need to have descriptors+adapters under cmd/bgp, only for plugins/bgp (as you already have adapters prepared there).